Prolangle/wwwroot/js/app.js

```diff
@@ -25,7 +25,11 @@ async function shareResult(text) {
         return "Result copied to clipboard.";
     }
 
-    await navigator.share(shareData);
+    try {
+        await navigator.share(shareData);
+    } catch (e) {
+        console.warn(`Error occurred during share sheet: ${e}`);
```

Comment:
```suggestion
        console.warn('Error occurred during share sheet: ', e);
```

This way the JS console can render whatever the value is in a rich way as opposed to converting it to a string first.